make/modules/java.base/Lib.gmk

```diff
@@ -233,3 +233,23 @@ ifeq ($(ENABLE_FALLBACK_LINKER), true)
 
   TARGETS += $(BUILD_LIBFALLBACKLINKER)
 endif
+
+################################################################################
+
+ifeq ($(call isTargetOs, linux)+$(call isTargetCpu, x86_64)+$(INCLUDE_COMPILER2), true+true+true)
+  $(eval $(call SetupJdkLibrary, BUILD_LIB_X86_64, \
+      NAME := x86_64, \
+      OPTIMIZATION := HIGH, \
+      CFLAGS := $(CFLAGS_JDKLIB), \
+      CXXFLAGS := $(CXXFLAGS_JDKLIB), \
+      LDFLAGS := $(LDFLAGS_JDKLIB) \
+          $(call SET_SHARED_LIBRARY_ORIGIN), \
+      LDFLAGS_linux := -Wl$(COMMA)--no-as-needed, \
```

Comment:
This is set by default since JDK-8314554.

Reply:
Thanks Erik. Will update Lib.gmk accordingly.

Reply:
Based on your suggestion, please see the LDFLAGS_linux removed in the latest commit.